.github/workflows/e2e-upgrade.yml

```
@@ -212,6 +289,12 @@ jobs:
         with:
           azure_credentials: ${{ secrets.AZURE_E2E_IAM_CREDENTIALS }}
 
+      - name: Download CLI
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: cli
```

```suggestion
          name: constellation
```
The following steps assume the CLI is at `build/constellation` not `build/cli`